### PR TITLE
1040 search function

### PR DESCRIPTION
**Pull request by PoojaKumari05**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

**Comment by bms63** [comment]
Taking this for a spin - I see that the References are aligned differently then the current version of the website.  Can you see if it is possible to align them to be the same?
image

This is so cool that we can search so easily!!
image

I put `search: exclude: ['news/index.html']` in the yml to ignore the changelog.  I think this would be helpful as I don't think a lot of people want to search the changelog/news.

image

File: renv.lock [hunk 1]
```diff
@@ -23,13 +23,6 @@
       "Repository": "CRAN",
       "Hash": "0e120603cc57e4f1d741f739aa8147ba"
     },
-    "MASS": {
-      "Package": "MASS",
-      "Version": "7.3-51.5",
-      "Source": "Repository",
-      "Repository": "CRAN",
-      "Hash": "9efe80472b21189ebab1b74169808c26"
-    },
```
Comment (bms63) [inline renv.lock:32]: How come this was removed?

**Comment by bms63** [comment]
@PoojaKumari05  The change in the look for the Reference Tab was determined to not to be an issue.  Rather have better search capability then the exact same layout.  Are you still able to push this along?

**Comment by PoojaKumari05** [comment]
> @PoojaKumari05 The change in the look for the Reference Tab was determined to not to be an issue. Rather have better search capability then the exact same layout. Are you still able to push this along?

@bms63 Hi Ben, as per the new bootstrap 5 update, the font size of the themes are larger than before, this is the reason the descriptions in References is falling to next line. If this is good to go with, I think we can closed this PR.
Capture

**Review by thomas-neitmann — APPROVED** [review]

**Pull request merged**